test/parallel/test-child-process-fork-regr-gh-2847.js

```diff
@@ -60,11 +60,8 @@ const server = net.createServer(function(s) {
       send(function(err) {
         // Ignore errors when sending the second handle because the worker
         // may already have exited.
-        if (err) {
-          if ((err.message !== 'channel closed') &&
-             (err.code !== 'ECONNREFUSED')) {
-            throw err;
-          }
+        if (err && err.message !== 'Channel closed') {
+          throw err;
```

Comment:
Note: the thrown error should never have that error code `ECONNREFUSED` as far as I can see.